cartridge/argparse.lua

```diff
@@ -170,6 +171,7 @@ local box_opts = {
     wal_queue_max_size       = 'number', -- **number**
     wal_dir_rescan_delay     = 'number', -- **number**
     wal_cleanup_delay        = 'number', -- **number**
+    wal_ext                  = 'json',   -- **json**
```

Comment:
It's worth to support log_modules (https://github.com/tarantool/tarantool/issues/8051) and metrics (https://github.com/tarantool/tarantool/issues/8494) as well